kinder/ci/tools/update-workflows/config.yaml

```diff
@@ -118,10 +124,13 @@ jobGroups:
     kubeadmVersion: latest
   - kubernetesVersion: -1
     kubeadmVersion: 0
+    ignorePreflightErrors: KubeletVersion
   - kubernetesVersion: -2
     kubeadmVersion: -1
+    ignorePreflightErrors: KubeletVersion
   - kubernetesVersion: -3
     kubeadmVersion: -2
+    ignorePreflightErrors: KubeletVersion
```

Comment:
`ignorePreflightErrors: KubeletVersion` is not needed, because the kubeletversion == kubernetesversion?